### PR TITLE
Add route scenario

### PR DESCRIPTION
**Pull request by Uvindu96**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: scenarios/scenario-18/README.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+## Scenario 17 - Expose an API using Openshift Route
```
Comment (pubudu538) [inline scenarios/scenario-18/README.md:1]: This is scenario 18.

File: scenarios/scenario-18/swagger.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+x-wso2-production-endpoints: hello-world-service
+x-wso2-mode: privateJet
+paths:
+  /hello/node:
+    get:
+      responses:
+        "200":
+          description: ""
+      x-auth-type: None
+      x-throttling-tier: Unlimited
```
Comment (pubudu538) [inline scenarios/scenario-18/swagger.yaml:32]: We dont need to set these to extensions.. Let's remove those.

File: scenarios/scenario-18/README.md (hunk 1 above)
Comment (Uvindu96) [reply scenarios/scenario-18/README.md]: updated to 18

File: scenarios/scenario-18/swagger.yaml (hunk 2 above)
Comment (Uvindu96) [reply scenarios/scenario-18/swagger.yaml]: extra extensions removed

**Pull request merged**